engine/src/main/java/org/terasology/engine/core/modes/StateLoading.java

```diff
@@ -234,7 +234,7 @@ private void popStep() {
         current = null;
         if (!loadProcesses.isEmpty()) {
             current = loadProcesses.remove();
-            logger.debug("{}", current.getMessage());
+            logger.atDebug().addArgument(() -> current.getMessage()).log("{}");
```

Comment:
In almost all cases constant string return so impact neglectable
```suggestion
            logger.debug("{}", current.getMessage()); //NOPMD
```